README.md

```diff
@@ -24,19 +26,20 @@ There are a few install options that can be set via environment variables:
 language: python
 matrix:
   include:
-    - python: "2.7"
-    - python: "pypy"
-      env: PYPY_VERSION="5.3.1"
+    - python: '2.7'
+    - python: '3.5'
+    - python: pypy
+      env: PYENV_VERSION="pypy-5.4.1" PYENV_VERSION_STRING="PyPy 5.4.1"
 cache:
   - pip
   - directories:
-    - $HOME/.pyenv_cache
+    - ~/.pyenv_cache
 
 before_install:
   - |
-      if [[ -n "$PYPY_VERSION" ]]; then
-        wget https://github.com/praekeltfoundation/travis-pypy/releases/download/0.1.0/setup-pypy.sh
-        source setup-pypy.sh
+      if [[ -n "$PYENV_VERSION" ]]; then
+        git clone https://github.com/praekeltfoundation/travis-pyenv.git ~/travis-pyenv
+        source ~/travis-pyenv/setup-pyenv.sh
```

Comment:
Only reason is that there isn't a released version yet and I didn't want to include a broken command.